frontend/src/routes/+page.svelte

```diff
@@ -1,7 +1,23 @@
-<script lang="ts">
-    import data from "$lib/data";
-</script>
-
-<div class="content">
-    <p>Some data is: {data}</p>
+<div class="block" style="margin-top: 15vh; margin-left: 10vh;">
+    <h1 class="title is-1">Whatever topic.</h1>
+    <h2 class="subtitle is-3">And maybe a little elaboration.</h2>
+    <p>Testing out text color.</p>
+    <a class="button is-large is-info" href="/"><strong>Get started</strong></a>
 </div>
+
+<div class="bg-img" />
+<slot />
+
+<style>
+    .bg-img {
+        position: absolute;
+        height: 100%;
+        width: 100%;
+        top: 0;
+        background-image: url(home.jpeg);
```

Comment:
This breaks the static adapter, and so doesn't render the image when built. To fix, use `url("/home.jpeg")`

Reply:
Done :)